package.json

```diff
@@ -113,6 +113,16 @@
           "type": "string",
           "default": "meson",
           "description": "Specify the meson executable to use"
+        },
+        "mesonbuild.linters.muon.enabled": {
```

Comment:
maybe `mesonbuild.linter` is a better option name with Muon being the default? Presumably more linters could be added in the future.

Alongside this option could be `mesonbuild.linting.enabled` defaulting to true.